### PR TITLE
Improvements and bug fixes

### PR DESCRIPTION
**Pull request by caike08**
Hey buddy,

I'm sending some changes I've done:
- Fixed floating labels on home screen
- Mark current tab as active (with active-routes)
- Automatic CEP search
- On settings, user can switch tabs (it wasn't working)
- Fixed timeout code, now it loads just after a page is rendered
- "Voltar" button added, yet it's not working

screenshot from 2016-09-25 18 37 56
screenshot from 2016-09-25 19 44 23
untitled

### CODE REVIEW

File: client/html/pages/panel/settings.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
+<template name="settings">
```
Comment (MarceloBorgesP) [inline client/html/pages/panel/settings.html:1]: Be careful with the indentation man, it looks like you add a 6 spaces indentation in here

File: client/html/pages/panel/settings.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+							</div><!--end .hbox-md -->
+						</div><!--end .card-tiles -->
+						<!-- END CONTACT DETAILS -->
+
+					</div><!--end .card -->
+				</div><!--end .section-body -->
+			</section>
+
+</template>
```
Comment (MarceloBorgesP) [inline client/html/pages/panel/settings.html]: Oh and the new line here

File: client/html/parts/menubar.html [hunk 3]
```diff
@@ -3,9 +3,12 @@
     <!-- MENUBAR CONTENT GOES HERE -->
     <div class="menubar-fixed-panel">
       <div>
-        <a class="btn btn-icon-toggle btn-default menubar-toggle" href="#" data-toggle="menubar">
+        <!--a class="btn btn-icon-toggle btn-default menubar-toggle" href="#" data-toggle="menubar">
```
Comment (MarceloBorgesP) [inline client/html/parts/menubar.html]: If we are not going to use you can remove it

File: imports/startup/client/materialadmin-load.js [hunk 4]
```diff
@@ -2,20 +2,36 @@ import {
...
 
-/* Runs fixFloatingLabels() before the route is completely changed */
-FlowRouter.triggers.enter([init]);
 
-function init() {
- Meteor.setTimeout(function() {
-    fixFloatingLabels();
-    reloadChosen();
- }, 10);
+Template.HomeLayout.rendered = function() {
```
Comment (MarceloBorgesP) [inline imports/startup/client/materialadmin-load.js:6]: I have no idea how it works man, but looks beautiful.. haha

File: imports/ui/components/pwd-strength.js [hunk 5]
```diff
@@ -1,42 +0,0 @@
-/* Password strength validator */
```
Comment (MarceloBorgesP) [inline imports/ui/components/pwd-strength.js:1]: Why did you remove it from here and added in the other archive? I think that if we can separate this logic into another file it is better organized

Comment (caike08) [reply imports/ui/components/pwd-strength.js:1]: This way we just load one file when a form is called. As I had other calls loaded just on this part, I decided to move it here. Is it fine?

File: imports/startup/client/materialadmin-load.js (hunk 4 above)
Comment (caike08) [reply imports/startup/client/materialadmin-load.js:6]: As far as it works without timeout, it's fine :D

File: imports/ui/components/pwd-strength.js (hunk 5 above)
Comment (MarceloBorgesP) [reply imports/ui/components/pwd-strength.js:1]: Hmm right, well it is a solution for now, but I think we might separate it again in the future if we start adding to much logic in here
